Woof/Woof/Helper/AppAlert/AppAlert.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+/// Represents alerts which uses in application.
+enum AppAlert: String, CustomStringConvertible, CaseIterable {
+    /// Alert shown as confirmation of log out
+    case logOut = "Do you really want to log out?"
+
+    /// Alert shown as a result of an error
+    case error = "Error"
+
+    /// Text description of alert
+    var description: String {
+        rawValue
+    }
```

Comment:
As an allowed variant but up to you:
```suggestion
    /// Alert shown as confirmation of log out
    case logOut = "Do you really want to log out?"
    /// Alert shown as a result of an error
    case error = "Error"
    /// Text description of alert
    var description: String {
        rawValue
    }
```

Reply:
Got it, but I'm save the current version.